build.gradle

```diff
@@ -20,10 +20,12 @@ repositories {
 
 dependencies {
     compile name: 'touch-image-view'
+    compile 'com.android.support:support-v4:22.2.1'
     compile project(':owncloud-android-library')
     compile 'com.android.support:support-v4:22.2.1'
     compile 'com.jakewharton:disklrucache:2.0.2'
     compile 'com.android.support:appcompat-v7:22.2.1'
+    compile 'com.getbase:floatingactionbutton:1.10.1'
```

Comment:
Sweet, tan we can keep the lib for now. Whenever a speeddial implementation would be available within googles support lib like "design" I would than refactor it and remove the third party lib... continuous improvement so to say :smiley: